src/Store/ExtensionClientStoreConnector.re

```diff
@@ -70,9 +124,14 @@ let start = (extensions, setup: Core.Setup.t) => {
     );
   };
 
-  let onRegisterSuggestProvider = (sp: LanguageFeatures.SuggestProvider.t) => {
+  let onRegisterSuggestProvider = (client, sp) => {
```

Comment:
`sp` is too abbreviated I think. I would just use `provider` here, since the context strongly suggests (^^) that a `SuggestProvider` is the primary kind of `provider` here.